tests/find_iocs_cases/domains.py

```diff
@@ -7,11 +7,9 @@
     param(
         "https://asf.goole.com/mail?url=http%3A%2F%2Ffreasdfuewriter.com%2Fcs%2Fimage%2FCommerciaE.jpg&t=1575955624&ymreqid=733bc9eb-e8f-34cb-1cb5-120010019e00&sig=x2Pa2oOYxanG52s4vyCEFg--~Chttp://uniddloos.zddfdd.org/CBA0019_file_00002_pdf.zip",
         {
-            "domains": ["google.com", "freasdfuewriter.com", "uniddloos.zddfdd.org"],
+            "domains": ["asf.goole.com", "cba0019_file_00002_pdf.zip", "freasdfuewriter.com", "uniddloos.zddfdd.org"],
```

Comment:
`cba0019_file_00002_pdf.zip` should, ideally, not be caught here, but this will be fixed in #210 .